clis/teliod/src/daemon.rs

```diff
@@ -42,6 +43,13 @@ pub struct TelioTaskStates {
     interface_ip_address: Option<IpAddr>,
 }
 
+/// Create default features with direct connections enabled
+fn make_features() -> Features {
+    let mut features = Features::default();
+    features.direct = Some(FeatureDirect::default());
+    features
+}
```

Comment:
Did you consider using the `DefaultFeaturesBuilder`?

Reply:
I can not find anything called `DefaultFeaturesBuilder` in the code-base 😞

Reply:
Oh it's `FeaturesDefaultsBuilder`, i'll take a look :)

Reply:
Sorry, got the name mixed up :smile:

Reply:
Hmm looks like the previous method is a little bit more concise than:
```rust
    let features: Arc<FeaturesDefaultsBuilder> = Arc::new(FeaturesDefaultsBuilder::new());
    features.clone().enable_direct();
    features.build()
```

Reply:
Updated